src/eShopOnBlazor/Pages/Catalog/Delete.razor

```diff
@@ -29,15 +29,19 @@
             </dt>
 
             <dd>
-                @_item.CatalogBrand.Brand
+                @{
+                    Console.WriteLine(_item.CatalogBrand != null ? _item.CatalogBrand.Brand : String.Empty);
```

Comment:
Why are you writing this information to the console instead of outputting in the HTML?

Reply:
Updated as mentioned above.